rpc/rpc.go

```diff
@@ -179,7 +179,14 @@ func (g *Generator) genMethodCall(ctx *context, rpc *protobuf.RPC) ast.Expr {
 	}
 
 	if !isGenerated(rpc.Input) {
-		call.Args = append(call.Args, ast.NewIdent("in"))
+		var in ast.Expr = ast.NewIdent("in")
+		if !rpc.Input.IsNullable() {
+			in = &ast.UnaryExpr{
```

Comment:
please, use [StarExpr](https://golang.org/pkg/go/ast/#StarExpr) instead, it's more clear